x-pack/plugins/security_solution/public/management/common/url_routing/utils.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
```

Comment:
Some of these were copied over from the existing `routing.ts`. Once I refactor the existing `routing.ts` over to a new folder named `routing`, this `util.ts` module will likely hold all of the other utilities found there